doc/CONTRIBUTING.md

```diff
@@ -3,7 +3,15 @@ Contributing
 
 We are happy for you to contribute with new posteriors to the datasbase. just create a Pull Request with you new content to the posteriordb, the tests on the new content will be checked.
 
-Below are checklists depending on the what you want to contribute.
+Don't hesitate to do a PR. We use the  PR for discussion on new material!
```

Comment:
move this into the preceding paragraph. Explain the abbreviation PR if you use it as some users may not have heard of it. I think the correct term is "make a PR" but I may be wrong.